hotsos/core/host_helpers/cli/catalog.py

```diff
@@ -766,26 +812,68 @@ def __init__(self):
                          'ip_netns_exec_{namespace}_ip_address_show'),
                  FileCmd('sos_commands/networking/namespaces/{namespace}/'
                          'ip_netns_exec_{namespace}_ip_-d_address_show')],
+            'ovn_nbctl_list':
+                [OVNDBCTLListBinCmd(ctl_command='ovn-nbctl'),
+                 OVNDBCTLListBinCmd(ctl_command='ovn-nbctl',
+                                    prefix='openstack-hypervisor.'),
+                 OVNDBCTLListBinCmd(ctl_command='ovn-nbctl',
+                                    prefix='microovn.'),
+                 # sosreport: noalias
+                 OVNDBCTLListFileCmd(ctl_command='ovn-nbctl'),
```

Comment:
So I'll admit I'm a little lost here as the class want's a 'table' argument but none is provided here. But it is done in the OVNDBTable class below, so I guess it works? Other than that it looks Ok.

Reply:
so the way this works is that this class returns an object that is the command that is called() with table passed in as an arg e.g.

```OVNDBCTLListFileCmd(ctl_command='ovn-nbctl')(table='Connection')```

Reply:
Ack, as long as you're around to debug it lgtm :)